ambf_ros_modules/ambf_comm/scripts/ambf_rigid_body.py

```diff
@@ -77,14 +77,14 @@ def is_joint_idx_valid(self, joint_idx):
                   '\" outside valid range [0 - ' + str(n_jnts - 1) + ']')
             return False
 
-    def get_linear_velocity(self):
```

Comment:
This should be get_linear_vel as this method is supposed to return the vel of the body and not the joint